### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by Ch3LL**

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -67,19 +67,23 @@ repos:
...
   - repo: https://github.com/asottile/blacken-docs
     rev: v1.10.0
     hooks:
       - id: blacken-docs
         args: [--skip-errors]
         files: ^(docs/.*\.rst|src/saltext/salt_ext_heist/.*\.py)$
-        additional_dependencies: [black==20.8b1]
+        additional_dependencies:
+          - click<8.1.0
+          - black==21.12b0
```
Comment (s0undt3ch) [inline .pre-commit-config.yaml:86]: Same here.

**Review by s0undt3ch — APPROVED** [review]

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -67,19 +67,23 @@ repos:
         exclude: src/saltext/salt_ext_heist/version.py
 
   - repo: https://github.com/psf/black
-    rev: 20.8b1
+    rev: 21.12b0
```
[the remark this answers is not in the document]
Comment (Ch3LL) [reply .pre-commit-config.yaml:70]: It's not clear to me what I need to do. I did include click as an additional dependency

File: .pre-commit-config.yaml [hunk 3]
```diff
@@ -67,19 +67,23 @@ repos:
...
     hooks:
       - id: black
         args: [-l 100]
         exclude: src/saltext/salt_ext_heist/version.py
+        additional_dependencies:
+          - click<8.1.0
```
Comment (s0undt3ch) [inline .pre-commit-config.yaml:76]:
```suggestion
    rev: 22.3.0
    hooks:
      - id: black
        args: [-l 100]
        exclude: src/saltext/salt_ext_heist/version.py
```

File: .pre-commit-config.yaml (hunk 1 above)
Comment (s0undt3ch) [inline .pre-commit-config.yaml:86]:
```suggestion
        additional_dependencies:
          - black==22.3.0
```

**Review by s0undt3ch — APPROVED** [review]
You don't actually need to change if you don't want to, it was merely a fyi.
Anyway, here's the proper reply I should have done :grin:

**Comment by Ch3LL** [comment]
The latest version of black was causing issues for me, so I'll keep these changes as is and go ahead nad merge.

**Pull request merged**